### PR TITLE
add user change password in admin

### PR DESCRIPTION
**Pull request by kiddblizzard**

### CODE REVIEW

File: src/SMG/ManagerBundle/Controller/ManagersController.php [hunk 1]
```diff
@@ -166,4 +165,55 @@ public function patchUserEnableAction(User $user)
...
+    public function putUserPasswordAction(User $user, Request $request)
+    {
+        $this->throwIfClientNot('backend');
+
+        $requestData = $this->requestIsJsonWithKeysOrThrow(
+            $request,
+            ['new_password']
+        );
+
+        $user->setPlainPassword($requestData['new_password']);
```
Comment (thomas-barthelemy) [inline src/SMG/ManagerBundle/Controller/ManagersController.php:196]: that would mean anyone with a backend token (so anyone) can change the password of anyone else as long as they know this URL / the user ID. That's why I mentioned I could give the current user id + token to verify he has ROLE_ADMINPANEL

Comment (allan-simon) [inline src/SMG/ManagerBundle/Controller/ManagersController.php:196]: and it does not return any response

Comment (kiddblizzard) [inline src/SMG/ManagerBundle/Controller/ManagersController.php:196]: add test and only admin can update the password

**Comment by kiddblizzard** [comment]
change patch to put

File: src/SMG/ManagerBundle/Controller/ManagersController.php (hunk 1 above)
Comment (thomas-barthelemy) [inline src/SMG/ManagerBundle/Controller/ManagersController.php:196]: Still not returning anything on success

Comment (kiddblizzard) [inline src/SMG/ManagerBundle/Controller/ManagersController.php:196]: return 204

**Pull request merged**